packages/ng/core/date/native/native-date.adapter.ts

```diff
@@ -64,14 +64,14 @@ export class LuNativeDateAdapter extends ALuDateAdapter<Date> implements ILuDate
 			return false;
 		}
 		const groups = text.split(this._regex);
-		if (groups.length !== 3 && groups.length !== 2) {
+		if (groups.length !== 3 && groups.length !== 2 && groups.length !== 1) {
```

Comment:
Ugh this code is so weird (ans was like this before your PR). I'm guessing those lengths fit the `granularity` param so we could check for `granularity` before assessing their value ? 

like so

```suggestion
		if (
			(granularity === ELuDateGranularity.day && groups.length !== 3)
			|| (granularity === ELuDateGranularity.month && groups.length !== 2)
			|| (granularity === ELuDateGranularity.year && groups.length !== 1)
		) {
```

It would add some clarity / readability and be more accurate (I guess 😅)